recipe/common.php

```diff
@@ -18,13 +18,32 @@
 set('clear_paths', []);         // Relative path from deploy_path
 set('clear_use_sudo', true);    // Using sudo in clean commands?
 
+
+/**
+ * Composer options
+ */
+env('composer_action', function () {
+    return env('env') === 'prod' ? 'install' : 'update';
```

Comment:
I don't think using `composer update` is good action.
It will update your vendors and composer.lock file.